test.txt

```diff
@@ -0,0 +1 @@
+colin test
```

Comment:
better to add a space in the beginning of the line

Reply:
Add comments before coding should be better